src/components/Modals/FiatRamps/fiatRampProviders/mtpelerin.ts

```diff
@@ -0,0 +1,40 @@
+import { adapters } from "@shapeshiftoss/caip";
+import { FiatRampAction, FiatRampAsset } from "../FiatRampsCommon";
+
+
+const assetList = require("./mtpelerinAssets.json");
```

Comment:
This shouldn't live in web as a JSON.
It looks like this was copied from `https://api.mtpelerin.com/currencies/tokens` - you will need to
- implement a function that fetches this
- validates the list against the lib mapping
- parse it as a `FiatRampAsset[]` buy/sell list

It should look something like this:

https://github.com/shapeshift/web/blob/41bb36581344f5f9295186301d466c04522a9f9d/src/components/Modals/FiatRamps/config.ts#L37-L42